index.md

```diff
@@ -8,12 +8,12 @@ Welcome to the [WordPress Hosting Handbook](https://make.wordpress.org/hosting/h
 
 The [Hosting Team](https://make.wordpress.org/hosting/) works to improve the WordPress user experience across hosting environments through user collaboration and education.
 
-Everyone is welcome! This may be folks hosting WordPress, interested in learning more about hosting WordPress, or working with WordPress hosts, like plugin and theme authors.
+Everyone is welcome! This may include folks hosting WordPress, interested in learning more about hosting WordPress or folks working directly with WordPress hosts, such as plugin authors and system architects.
```

Comment:
Yes, I agree that's what the team looks like right now! I included theme authors because they often work with hosts, and it'd be great to have more folks involved / make it clear they belong.

I'm not opposed to changing it, though. And also to adding/noting specifically Systems Engineers / Architects / DevOps. I had been thinking that set was included in the "folks hosting WordPress" mention. Do you think that's a separate group of people? If so, I think something like what you've proposed here is 👍 

If they're mostly in the former, though, maybe something like this would work? "This includes folks hosting WordPress, including Systems Engineers, Systems Architects, or DevOps. It also includes folks that are interested in learning more about hosting WordPress, or work directly with WordPress hosts, such as plugin or theme authors."

Reply:
I would add both groups... Something like:

> Everyone is welcome! This may include folks hosting WordPress, interested in learning more about hosting WordPress or folks working directly with WordPress hosts, such System Engineers, System Architects or DevOps Engineers or Developers that are working on plugins or themes.

Reply:
Hmm, I think I might see where the confusion is. I intended  "folks hosting WordPress" to be "folks that are employed by a host," vs folks that are not employed by a host, but interact with a host, or have related interest.

I split it into two sentences to attempt to make it easier to follow, but maybe there's a way to phrase it so that it doesn't need to be.

Reply:
No, I think I understood what you meant, but my sentence was not that... perfect? 

* One Group: System Engineers, System Architects or DevOps Engineers.
* Other Group: Developers that are working on plugins or themes.

Reply:
ok. How about: 

```
Everyone is welcome! Participants may include folks who are interested in learning more about hosting WordPress, who work directly with WordPress hosts or who are employed by WordPress hosts such as support agents, engineers and developers.
```

Reply:
I like it! I really would prefer to mention plugin and theme authors directly, if we can, though -- because to me "developers" doesn't make it clear that folks that develop products for WordPress are included/welcome.

Reply:
@getsource 

> Everyone is welcome! Participants may include folks who are interested in learning more about hosting WordPress, who work directly with WordPress hosts or who are employed by WordPress hosts such as support agents, engineers and developers that work on plugins and themes.

Reply:
@pfefferle Looks good! Would add a comma here: "engineers, and"

Reply:
> Everyone is welcome! Participants may include folks who are interested in learning more about hosting WordPress, who work directly with WordPress hosts or who are employed by WordPress hosts such as support agents, engineers, and developers that work on plugins and themes.

Reply:
Also, apologies for drawing this change out, and thanks for everyone's patience. Hopefully it's clear, but the intention is to make sure we end up with something more inclusive at the end.

@kittenkamala If you think it's better/more honest, we could also do something like "Right now, the team includes (or is mostly) ..."  "We'd love to have more..."